composer.json

```diff
@@ -78,7 +78,8 @@
     "bin": ["bin/ibexabehat", "bin/ibexareport"],
     "config": {
         "allow-plugins": {
-            "composer/package-versions-deprecated": true
+            "composer/package-versions-deprecated": true,
+            "php-http/discovery": false
```

Comment:
This lie can be deleted, error regarding php-http/discovery should be fixed by now.

Reply:
It's true that it's no longer required, but Composer will keep asking about it when installing locally without this entry

I might have commented somewhere that it should be removed, but TBH that was a mistake by me - I'd keep this change

Reply:
I had the same concerns about it but Marek told me to leave it. Btw composer fix-cs did that change.